src/react-apps/applications/ux-editor/src/sagas/formDesigner/formDesignerSagas.ts

```diff
@@ -388,75 +406,78 @@ function* createRepeatingContainer(
-      yield call(createRepeatingContainer, createdContainerId, elementId, newContainer, newContainerId);
-    }
-    if (conditionalRenderingRules[elementId]) {
-      // We have a relevant condtional rendering rule that has to be copied for the newly created element
-      const condtionalRuleInfo = conditionalRenderingRules[elementId];
-      const newConditionalRuleId: string = uuid();
-      const newCondtitionalRule: any = {
-        ...serviceConfigurations.conditionalRendering[condtionalRuleInfo.conditionalRenderingId],
-      };
-      const selectedFieldsObject: any = {};
-      selectedFieldsObject[newConditionalRuleId] = createdElementId;
-      newCondtitionalRule.selectedFields = selectedFieldsObject;
-      const newConditionalRuleObject: any = {};
-      newConditionalRuleObject[newConditionalRuleId] = newCondtitionalRule;
-      yield call(
-        conditionalRenderingActionDispatcher.addConditionalRendering, newConditionalRuleObject);
+    let createdElementId: string;
+
+    for (const elementId of order[containerToCopyId]) {
+      if (components[elementId]) {
+        const createdConmponentId = uuid();
+        const newComponent = { ...components[elementId] };
+        createdElementId = createdConmponentId;
+        yield call(FormDesignerActionDispatchers.addFormComponentFulfilled,
+          newComponent, null, createdConmponentId, newContainerId);
+      } else if (containers[elementId]) {
+        const newContainer: ICreateFormContainer = {
+          repeating: containers[elementId].repeating,
+          index: (containers[elementId].index != null) ? (containers[elementId].index + 1) : null,
+          hidden: containers[elementId].hidden,
+          dataModelGroup: containers[elementId].dataModelGroup,
+        };
+
+        // Recursive call, since containers can have sub-containers.
+        const createdContainerId = uuid();
+        createdElementId = createdContainerId;
+        yield call(createRepeatingContainer, createdContainerId, elementId, newContainer, newContainerId);
+      }
+      if (conditionalRenderingRules[elementId]) {
+        // We have a relevant condtional rendering rule that has to be copied for the newly created element
+        const condtionalRuleInfo = conditionalRenderingRules[elementId];
+        const newConditionalRuleId: string = uuid();
+        const newCondtitionalRule: any = {
+          ...serviceConfigurations.conditionalRendering[condtionalRuleInfo.conditionalRenderingId],
+        };
+        const selectedFieldsObject: any = {};
+        selectedFieldsObject[newConditionalRuleId] = createdElementId;
+        newCondtitionalRule.selectedFields = selectedFieldsObject;
+        const newConditionalRuleObject: any = {};
+        newConditionalRuleObject[newConditionalRuleId] = newCondtitionalRule;
+        yield call(
+          conditionalRenderingActionDispatcher.addConditionalRendering, newConditionalRuleObject);
+      }
     }
+  } catch (err) {
+    console.error(err);
```

Comment:
Should we console.log errors?

Reply:
console.error writes the red text in the console. Log is for logging, error is for error. We do have this errorhandling component though. Can change this to send a rejectedAction and add the error to the error-component.

Reply:
Hmm. We do use `console.error` in the solution now.  It's useful for development, but might not be the best solution for production. Created issue #617 to come up with a general solution for how to handle technical errors.